test/logger_test.rb

```diff
@@ -455,9 +455,9 @@ class TestLogger < Hanami::Logger; end
           output =
             stub_stdout_constant do
               class TestLogger < Hanami::Logger; end
-              TestLogger.new.info('This is a message with spaces')
+              TestLogger.new.info("This is a message with quotes \" ' and with spaces")
```

Comment:
@AlfonsoUceda I was joking :wink: 

Of course it passes, because I used an alternative Ruby syntax.
My point is about the output that is generated by that escaping.

Let me try to explain with code. If in the code you add the following line:

```ruby
File.open("log", "w+") { |f| f.write(output) }
```

Then if you look at `log`, here's the result:

```
app=Hanami severity=INFO time=1988-09-01T00:00:00+00:00 message="This is a message with quotes " ' and with spaces"
```

If you look at the message, is interrupted: the first `"` matches with the one in the middle of the sentence. So the message is: `This is a message with quotes `, while the remaining part `' and with spaces"` would get a parser crazy.